release_notes_generator/generator.py

```diff
@@ -125,3 +126,35 @@ def generate(self) -> Optional[str]:
         )
 
         return release_notes_builder.build()
+
+    def get_latest_release(self, repo: Repository) -> Optional[GitRelease]:
+        """
+        Get the latest release of the repository.
+
+        @param repo: The repository to get the latest release from.
+        @return: The latest release of the repository, or None if no releases are found.
+        """
+        if ActionInputs.is_from_tag_name_defined():
+            logger.info("Getting latest release by from-tag name %s", ActionInputs.get_tag_name())
+            rls = self._safe_call(repo.get_release)(ActionInputs.get_from_tag_name())
```

Comment:
Also suggesting adding a type hint. In this context, it make the code easier to read.

Reply:
Fixed in commit 3bd9b0f89377c110b90a3c003373aa1e44cc7930.